src/abstractdataframe/abstractdataframe.jl

```diff
@@ -34,8 +34,6 @@ The following are normally implemented for AbstractDataFrames:
 * [`allowmissing`](@ref) : add support for missing values in columns
 * [`allowmissing!`](@ref) : add support for missing values in columns in-place
 * `similar` : a DataFrame with similar columns as `d`
-* `filter` : remove rows
-* `filter!` : remove rows in-place
```

Comment:
Likewise, I'd keep this as these are implemented.

Reply:
Actually - after thinking about it I would remove this list. It is hard to maintain (contains e.g. `length` now and does not list many functions that are implemented).

Reply:
it also contains `convert`.